### ISSUE TITLE
Refactor settings to use dconf

### ISSUE THREAD

**Issue by Sjoerd1993** [Sjoerd1993/Graphs#389]
I've been told using dconf rather than json files is the preferred way to store application settings in GNOME. 
https://docs.gtk.org/gio/class.Settings.html

Perhaps not directly with the upcoming release (depends a bit on when we decide to release, we can wait a bit with 1.7.0), but we should probably migrate to this logic in the future.

**Comment by cmkohnen** [comment]
I don't see a reason not to include this in the 1.7.0 release, we don't have any blatant issues with the current release, that need to be adressed

**Comment by Sjoerd1993** [comment]
Yeah, hence the "we can wait a bit with 1.7.0" remark. It fully depends on what we decide the scope of 1.7.0 to be.

There's some bigger changes coming up, and refactoring the settings will break quite a bit of stuff (e.g. project files). So we can either release 1.7.0 relatively soon before all those changes, so that e.g. the view changer is out in production. Or we wait with 1.7.0 until things are relatively stable again (at least until we've addressed most, if not all, of the GNOME circle feedback). 

I was initially planning on releasing 1.7.0 relatively soon to get some of the features out, ahead of the upcoming changes, that have been implemented since last major release (well, mainly the view switcher). But since there's no pressing bugs, or people screaming for specific features, I am also fine with waiting a while. If some major bug does turn up, we can always write a patch based on the 1.6.1 branch and push 1.6.2.
